src/vm/ecalllist.h

```diff
@@ -373,10 +373,8 @@ FCFuncEnd()
 
 FCFuncStart(gTypedReferenceFuncs)
     FCFuncElement("InternalToObject", ReflectionInvocation::TypedReferenceToObject)
-#ifndef FEATURE_CORECLR
     FCFuncElement("InternalSetTypedReference", ReflectionInvocation::SetTypedReference)
```

Comment:
These you've now exposed won't actually get called yet right? I suppose there is no harm to un-ifdeffing them in advance.